gcodestat.c

```diff
@@ -259,7 +261,6 @@ int main(int argc, char** argv) {
    static struct option long_options[] = {
       {"help", no_argument, NULL, 'h'},
       {"quiet", no_argument, NULL, 'q'},
-      {"show_time", no_argument, NULL, 'Q'},

```

Comment:
Why? you have option to run with or without "shot time" why remove the possibility? it was not there because noone needed it?